backend/src/main/java/com/htecgroup/skynest/service/CompanyService.java

```diff
@@ -12,5 +13,7 @@ public interface CompanyService {
 
   CompanyResponse addCompany(@Valid @UniqueCompany CompanyAddRequest companyAddRequest);
 
+  CompanyResponse editCompany(CompanyEditRequest companyEditRequest);
```

Comment:
Valid annotation?

Reply:
No need, this request only contains name, address and phone number, company Id is taken from the session token. 
Name, address and phone number are validated in the controller method